tests/conftest.py

```
@@ -8462,20 +8463,39 @@ def factory(namespace=None, encrypted=False, use_cluster_capacity=False):
         return (
             vm_list_default_compr,
             vm_list_agg_compr,
             sc_obj_def_compr,
             sc_obj_aggressive,
         )
+    def teardown():
+        log.info("Starting multi_cnv_workload teardown")
+
+        for vm in created_vms:
+            try:
+                log.info(f"Deleting VM {vm.name}")
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)